### DISCUSSION TITLE
Undefined index: guard_name error

### DISCUSSION THREAD

**Discussion by mostafizurhimself** [spatie/laravel-permission#1901] [Q&A]
When I call Auth::user()->hasPermissionTo('permission-name) it causes :
ErrorException
Undefined index: guard_name

Here is the screenshot:
![image](https://user-images.githubusercontent.com/42089253/132349196-7d3dcab7-d884-4b81-aeb3-f2b0a8d70873.png)

**Comment by erikn69** [comment]
You have to give more información, i don't know your environment, laravel version, package version, etc

Did you try `php artisan permission:cache-reset`?

Also on that image i don't see line 238, only till 235
